Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -193,12 +194,17 @@ async function run() {
                 await kuduUtility.runPostDeploymentScript(publishingProfile, kuduWorkingDirectory, scriptType, inlineScript, scriptPath, takeAppOfflineFlag);
             }
         }
+
         await updateWebAppConfigDetails(endPoint, webAppName, resourceGroupName, deployToSlotFlag, slotName);
     }
     catch (error) {
         isDeploymentSuccess = false;
         tl.setResult(tl.TaskResult.Failed, error);
     }
+
+    // Add release annotation for App Services which has an Application Insights resource configured to it.
+    await addReleaseAnnotation(endPoint, webAppName, webAppSettings, isDeploymentSuccess);
```

Comment:
Add try-catch for better reliability of task.
Will it abort the task when `azureRESTUtility.getApplicationInsightsResources` throws error?

Reply:
Yes it will. Thanks. I will make the changes.